### DISCUSSION TITLE
v5 website and the docs is very nice 🎉🥳

### DISCUSSION THREAD

**Discussion by masterbater** [feathersjs/feathers#2971] [General]
The new v5 seems looking good  and
been waiting for a nodejs typescript framework that meet this criteria
- No decorators, no modules or DI for small or medium project

Hope you could add this in the docs on how to integrate this, its been rising popularity in full typescript stack
- Supports Prisma for orm
- Supports zod as a validation library
- TRPC Example

**Comment by daffl** [comment]
Thank you for the feedback! To your questions:

- There is a [feathers-prisma](https://www.npmjs.com/package/feathers-prisma) adapter but it might need contributors to update everything to the latest release. I'd be curious to see what the SQL and MongoDB functionality that comes with Feathers 5 is missing. It is definitely more performant and leaner than the Prisma ORM.
- Feathers schema has been built to be validation library independent. Support for Zod has definitely been on the radar (/cc @DaddyWarbucks) but we went with JSON schema and TypeBox (a JSON schema builder) first because JSON schema is a standard that is supported by many tools like the OpenAPI specification and more.
- This might be an option as a new transport but I'm not sure if it adds anything that isn't already possible. A generated v5 application comes with a [fully typed client](https://feathersjs.com/guides/cli/client.html) that you can install in your React/Vue/Angular etc. TypeScript project and get the always up to date types and services. It's the same client that Feathers had for over 6 years, but with added type information. There is no custom protocol, just the standard Feathers API interface. It is faster than e.g. GraphQL (which needs to parse a custom DSL) and there is no bundle size overhead since the types will be removed at compile time.
